docker-compose.yml

```diff
@@ -195,29 +359,92 @@ services:
+                ip link set eth0.10 up && \
+                ip addr add 10.10.10.11/24 dev eth0.10 && sleep infinity'
+
+  host3-leaf2:
+    image: ubuntu
+    cap_add:
+      - NET_ADMIN
+    networks:
+      n4htoleafbn2:
+    command: |
+      sh -x -c 'apt-get update && apt-get install iproute2 -y && apt-get install iputils-ping -y && sleep 30 && \
+                ip link add link eth0 name eth0.30 type vlan id 30 && \
+                ip link set eth0 up && ip addr flush dev eth0 && \
+                ip link set eth0.30 up && \
+                ip addr add 30.30.30.31/24 dev eth0.30 &&
+                ip route replace default via 30.30.30.1 && sleep infinity'
+
   opi-test:
-    image: alpine:3.18
-    network_mode: service:leaf1
+    image: ubuntu
+    cap_add:
+      - NET_ADMIN
+    networks:
+      n2htoleaf1:
+      n3htoleafbn1:
+      n5h1tol1r:
+      n6h1tol1y:
     depends_on:
+      - spine1
       - leaf1
       - leaf2
       - bleaf
-      - spine1
+      - host2-leaf2
+      - host3-leaf2
       - opi-evpn-bridge
       - testgrpc
-    # L2 VXLAN - VLAN10 stretched with VNI10 to leaf2 from leaf1 (ping 10.10.10.3)
-    # L3VXLAN Asymmetric IRB - VLAN20 stretched with VNI20 to leaf2 from leaf1 (ping 20.20.20.3) on blue VRF
-    # L3VXLAN Symmetric IRB - L3VNI 100 is used for borderleaf connectivity to reach the internet
-    command: sh -x -c 'sleep 60 && ping -c 3 5.5.5.5 -I br100 && ping -c 3 10.10.10.3 && ping -c 3 20.20.20.3 -I br20'
+    # HOST1
+    # L2 VXLAN - VLAN10 stretched with VNI10 to leaf2 from leaf1, ping Host2 IP from Host1
+    # L3VXLAN Asymmetric IRB:
+    #   VLAN20 & VLAN30 stretched with VNI20 & VNI30 repectively to leaf2 from leaf1 via blue VRF, ping Host3 IP from Host1
+    # L3VXLAN Symmetric IRB:
+    #   Ping to Bleaf internet connectivity IPs from Host1 via red and yellow VRFs from Leaf1
+    command: |
+      sh -x -c 'apt-get update && apt-get install iproute2 -y && apt-get install iputils-ping -y && sleep 30 && \
+                ip link add link eth0 name eth0.10 type vlan id 10 && \
+                ip link set eth0 up && ip addr flush dev eth0 && \
+                ip link set eth0.10 up && \
+                ip addr add 10.10.10.10/24 dev eth0.10 && \
+                ping -c 3 10.10.10.11 && \
+                ip link add link eth1 name eth1.20 type vlan id 20 && \
+                ip link set eth1 up && ip addr flush dev eth1 && \
+                ip link set eth1.20 up && \
+                ip addr add 20.20.20.20/24 dev eth1.20 && \
+                ip route replace default via 20.20.20.1 && \
+                ping -c 3 30.30.30.31 && ping -c 3 5.5.5.5 && \
+                ip link add link eth2 name eth2.40 type vlan id 40 && \
+                ip link set eth2 up && ip addr flush dev eth2 && \
+                ip link set eth2.40 up && \
+                ip addr add 40.40.40.40/24 dev eth2.40 && \
+                ip route replace default via 40.40.40.1 && \
```

Comment:
instead of changing default `ip route replace default via 40.40.40.1` can you just use `ping -I interface 6.6.6.6` ?

Reply:
No, if we use ping 6.6.6.6 with src intf, the ARP will be sent for 6.6.6.6 from host, that's not correct.

Reply:
I see, thanks